src/strapi/index.ts

```diff
@@ -76,12 +76,39 @@ export const normalize = (
       total?: number
       limit?: number
       start?: number
+      pages?: number
+      page?: number
+      hasNextPage?: boolean
+      hasPrevPage?: boolean
     }
   }
   data: Array<unknown> | unknown
 } => {
+  let pages
+  let page
+  if (content?.meta?.pagination?.total && content?.meta?.pagination?.limit) {
+    pages = Math.ceil(content.meta.pagination.total / content.meta.pagination.limit)
+  }
+
+  if (content?.meta?.pagination?.start && content?.meta?.pagination?.limit) {
+    page = content.meta.pagination.start / content.meta.pagination.limit + 1
+  }
+
+  const hasNextPage =
+    content?.meta?.pagination?.limit + content.meta.pagination.start <
```

Comment:
if 'content?.meta?.pagination?.limit' results in undefined, the `hasNextPage` will always be false....am I wrong? Should we handle this kind of cases? what if `content?.meta?.pagination?.total` is undefined?

Reply:
I was just asking so that the algorithm does not fail in some sort of unexpected state...